app/frontend/src/components/History.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-container>
+    <v-skeleton-loader
+      type="table"
+      :types="{
+        table: 'table-thead, table-tbody',
+        'table-row': 'table-cell@3',
+        'table-thead': 'heading@3',
+      }"
+      :loading="!loaded"
+      transition="scale-transition"
+    >
+      <v-container>
+        <p class="text-right">Application <strong>{{ acronym }}</strong> created on: {{ formatDate(acronymCreated) }}</p>
+        <v-alert v-if="errorLoading" type="error" tile dense>
+          An error occurred while loading history, please refresh and try again
+        </v-alert>
+        <v-simple-table class="getok-simple-table">
+          <template v-slot:default>
```

Comment:
Can't remember, but if it's the default v-slot, you might be able to omit the `v-slot:default` part.
Also wondering if it would be worthwhile putting in an "empty" message when no records show on table instead?

Reply:
Yeah slot not needed.
No `no-data` prop on the v-simple-tables we use here so `v-if`'d a message in